src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -118,23 +190,94 @@ const GovernanceReviewTeam = () => {
             })}
           >
             <div className="grid-container overflow-auto">
-              <dl className="easi-grt__status-info text-gray-90">
-                <dt className="text-bold">{t('status.label')}</dt>
-                &nbsp;
-                <dd
-                  className="text-uppercase text-white bg-base-dark padding-05 font-body-3xs"
-                  data-testid="grt-status"
-                >
-                  {isIntakeClosed(systemIntake.status)
-                    ? t('status.closed')
-                    : t('status.open')}
-                </dd>
-                {systemIntake.lcid && (
-                  <>
-                    <dt>{t('intake:lifecycleId')}:&nbsp;</dt>
-                    <dd data-testid="grt-lcid">{systemIntake.lcid}</dd>
-                  </>
-                )}
+              <dl className="easi-grt__status-group">
+                <div className="easi-grt__status-info text-gray-90">
+                  <dt className="text-bold">{t('status.label')}</dt>
+                  &nbsp;
+                  <dd
+                    className="text-uppercase text-white bg-base-dark padding-05 font-body-3xs"
+                    data-testid="grt-status"
+                  >
+                    {isIntakeClosed(systemIntake.status)
+                      ? t('status.closed')
+                      : t('status.open')}
+                  </dd>
+                  {systemIntake.lcid && (
+                    <>
+                      <dt>{t('intake:lifecycleId')}:&nbsp;</dt>
+                      <dd data-testid="grt-lcid">{systemIntake.lcid}</dd>
+                    </>
+                  )}
+                </div>
+                <div className="text-gray-90">
+                  <dd>
+                    <dt className="text-bold">
+                      {t('intake:fields.adminLead')}
+                    </dt>
+                    <dt className="padding-1">{getAdminLead()}</dt>
+                    <dt>
+                      <Button
+                        type="button"
+                        unstyled
+                        onClick={() => setModalOpen(true)}
+                      >
+                        {t('governanceReviewTeam:adminLeads.changeLead')}
+                      </Button>
+                      <Modal
+                        title={t(
+                          'governanceReviewTeam:adminLeads:assignModal.title'
+                        )}
+                        isOpen={isModalOpen}
+                        closeModal={() => setModalOpen(false)}
+                      >
+                        <h1 className="margin-top-0 font-heading-2xl line-height-heading-2">
+                          {t(
+                            'governanceReviewTeam:adminLeads:assignModal.header',
+                            { requestName: systemIntake.requestName }
+                          )}
+                        </h1>
+                        <AdminLeadContext.Provider
```

Comment:
I don't think using a `Context` here is strictly necessary. It would be simpler to just pass the props into the `AdminLeadRadioOption` directly since everything is happening in the same place:

```jsx
<RadioGroup>
  {grtMembers.map(k => (
    <AdminLeadRadioOption
      checked={ k === newAdminLead }
      label={ k }
      onChange={(e: React.ChangeEvent<HTMLInputElement>) => { setAdminLead(k) }}
    />
  ))}
</RadioGroup>
```

I think using Context is more useful when there is a greater distance between where the values are being set and where they are used.

Reply:
I tried to make this work without context and state but could not figure it out. I knew I was probably missing something. thanks, I will make this change.

Reply:
and for the rest above I totally missed the red alert, will add that. Good catch on the radio option persisting that behavior is probably the opposite of what it should be, I will fix it.